unified-scheduler-pool/src/lib.rs

```diff
@@ -1013,29 +1004,14 @@ impl<S: SpawnableScheduler<TH>, TH: TaskHandler> ThreadManager<S, TH> {
                 let mut state_machine = unsafe {
                     SchedulingStateMachine::exclusively_initialize_current_thread_for_scheduling()
                 };
-                let mut result_with_timings = initialized_result_with_timings();
 
+                // The following loop maintains and updates ResultWithTimings as its
+                // externally-provieded mutable state for each session in this way:
```

Comment:
typo:

```suggestion
                // externally-provided mutable state for each session in this way:
```

Reply:
err, I'll merge this pr as-is. There's another pr for cosmetics fixes...

Reply:
all good, that's why I approved 😉